seaclouds-catalog.bom

```diff
@@ -253,6 +289,7 @@ brooklyn.catalog:
             install.latch: $brooklyn:component("planner").attributeWhenReady("service.isUp")
           - type: seaclouds-planner
             install.latch: $brooklyn:component("sla-core").attributeWhenReady("service.isUp")
+            install.latch: $brooklyn:component("discoverer").attributeWhenReady("service.isUp")
```

Comment:
I am tying to understand how to write the script since I couldn't find some information in brooklyn documentation and blueprint examples. Could you please support me in this process?